deploy/criReturnStepFunction.asl.json

```diff
@@ -92,7 +92,7 @@
         },
         {
           "Variable": "$.lambdaResult.journey",
-          "StringEquals": "/journey/next",
+          "StringEquals": "/journey/ci-score-not-breaching",
```

Comment:
Issue with this is if /journey/next can come from RetrieveCriCredential or ValidateOauthCallback